apps/azure-functions-api/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "azure-functions-api",
+  "version": "0.0.1",
+  "description": "Azure Function REST API Template",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "clean": "shx rm -rf dist",
+    "clean:generated": "shx rm -rf ./src/generated",
+    "build": "tsc",
+    "build:watch": "yarn build -w",
+    "typecheck": "yarn build --noemit",
+    "extensions:install": "func extensions install",
+    "start": "dotenv -e .env func start --javascript",
+    "start:watch": "yarn start watch",
+    "lint": "yarn lint:check --fix",
+    "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+    "lint:api": "sh api-lint.sh",
+    "api:bundle": "bundle-api-spec -i api/internal.yaml.template -o api/internal.yaml -V $npm_package_version",
+    "generate": "turbo run generate:definitions-internal",
+    "generate:definitions-internal": "shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal "
+  },
+  "devDependencies": {
+    "@pagopa/eslint-config-node": "*",
+    "@pagopa/typescript-config-node": "*",
+    "@types/express": "^4.17.12",
+    "@types/node": "~20.10.0",
+    "dependency-check": "^4.1.0",
+    "dotenv-cli": "^7.4.1",
+    "eslint": "^8.57.0",
```

Comment:
we should update `eslint` to its latest version

Reply:
Se https://github.com/pagopa/dx-typescript/pull/4#discussion_r1570495396